automation/catch-uncommitted-output

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Can we rename the file to `catch-uncommitted-output.sh` to give a hint on its type?